File: Data/Interfaces/DataTransferObjects/WebServiceDTO.cs
using Newtonsoft.Json;

namespace Data.Interfaces.DataTransferObjects
{
	public class WebServiceDTO
	{
        [JsonProperty("id")]
        public int Id { get; set; }

        [JsonProperty("name")]
        public string Name { get; set; }

        [JsonProperty("iconPath")]
        public string IconPath { get; set; }
	}
}

File: Hub/HubWeb/Scripts/app/model/WebServiceDTO.ts
module dockyard.model {
	export class WebServiceDTO {
		id: number;
		name: string;
		iconPath: string;

		constructor(id: number, name: string, iconPath: string) {
			this.id = id;
			this.name = name;
			this.iconPath = iconPath;
		}
	}
}